docs/api-reference/react/deckgl.md

```diff
@@ -106,23 +115,47 @@ It is possible to use JSX syntax to create deck.gl layers as React children of t
 For more information on this syntax and its limitations, see [Using deck.gl with React](/docs/get-started/using-with-react.md) article.
 
 
-#### Auto-Positioned Children
+#### JSX views
+
+It is possible to use JSX syntax to create deck.gl views as React children of the `DeckGL` React components, instead of providing them as ES6 class instances to the `views` prop.
+
+```jsx
+  <DeckGL initialViewState={...viewState} layers={layers} >
+    <MapView id="map" width="50%" controller={MapController} />
+    <FirstPersonView width="50%" x="50%" fovy={50} />
+  <DeckGL />
+```
+
+For more information on this syntax, see [Using deck.gl with React](/docs/get-started/using-with-react.md) article.
+
+
+#### Position Children in Viewport
 
 To make it easy to use React components in combination with deck.gl views (e.g. to place a base map under a view, or add a label on top of a view), deck.gl can make such components automatically adjust as that view is added, removed or resized.
 
-`DeckGL` classifies any top-level children (`props.children`) that have a `viewId` property as "view base components". It will perform special processing on them as follows:
+`DeckGL` accepts top-level children (`props.children`) that are instances of the `View` class. Any children nested under a `<View>` are automatically positioned as such:
+
+* It is offset to be relative to the deck.gl view with the corresponding view id.
+* Its parent DOM container is resized to match the extent of the deck.gl view with the corresponding view id.
+* If it is a callback function, it receives view state properties (`width`, `height`, `viewState` etc).
 
-* It resizes and repositions any `viewId` children to precisely match the extends of the deck.gl view with the corresponding id.
-* It automatically hides any `viewId` children whose id is not matched by any current deck.gl view.
-* It injects view state properties (`longitude`, `latitude` etc).
-* Also injects the `visible: viewport.isMapSynched()` prop to hide base maps that cannot display per the current view state parameters.
+```jsx
+  <DeckGL {...viewState} layers={layers} >
+    <MapView id="map" width="50%" >
+      <div>Map</div>
+    </MapView>
+    <FirstPersonView width="50%" x="50%" fovy={50} >
+      <div>First Person</div>
+    </FirstPersonView>
+  <DeckGL />
```

Comment:
Unless I missed it above, maybe show a use of just `View` together with `views`?